### DISCUSSION TITLE
Random Announcer feature like SF4 and others - new feature sugestion

### DISCUSSION THREAD

**Discussion by Splashslash** [ikemen-engine/Ikemen-GO#883] [Feature Request]
Please this is NOT the same thing that announcer percombo. (ex: yes, good)

Mugen have the random feature for explods, sounds... exemple: 1910+random%9

And will be useful implement that in screenpacks for explods and sounds. Like announcer sounds.

In USF4 for exemple, for the same round, the announcer speaks differents(random) things. Each "round 1", "round 2" is different of the others from the others fights.

Exemple: for the third round the announcer can say "this is the last round, fight!" or "this is the final round!" or "its all  or nothing!" or "round 3, fight!"
With random feature you can make that easy.

Other point: in the middle of the fight you can define some sounds to be the announcer, like USF4, MvC3 etc.
Conditions to define? Ex: time: each 20 sec,30 sec. one random speak/or explod. Other condition: life, if life is 50%, 10% the announcer can say something random for become more dramatic/intense fight. 

And the player can desactive the announcer in option fight menu. (the same menu that can see the characters movelist).

**Comment by EgyLynx** [comment]
Sorry, that sound more it stage thinks than enige, but i not sure...

**Comment by Splashslash** [comment]
Depends of the focus and intention...

In this case no makes sense make that per stage. Its much more easy make for engine. The same thing for announcer combo words (good, yes, nice, crazy, ultra combo etc) Its much more easy make for engine. Theres no reason install all the combo words in each character. The announcer combo words is just a exemple. Its implemented.

**Comment by jefinhoyagami** [comment]
Will be a good improvement!

**Comment by EgyLynx** [comment]
Hmm maybe...
